src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_time_field/step_time_field.html

```diff
@@ -0,0 +1,136 @@
+<form
+  role="form"
+  ng-submit="stepTimeField.createIndexPattern()"
+  class="kuiPanel"
+>
+  <div class="kuiPanelHeader">
+    <div class="kuiPanelHeaderSection">
+      <h2 class="kuiPanelHeader__title">
+        Step 2 of 2: Optional settings
+      </h2>
+    </div>
+  </div>
+
+  <div class="kuiPanelBody">
+    <p class="kuiText kuiSubduedText kuiVerticalRhythm">
+      You've defined <strong>{{stepTimeField.indexPatternName}}</strong> as your index pattern. Now you can specify some optional settings before we create it.
+      <small>
+        <a
+          class="kuiLink"
+          ng-click="stepTimeField.toggleAdvancedOptions()"
+        >
```

Comment:
Sounds good. I'll wait to hear back from @snide on the latter, but I'll fix up the former.